#goal definition
float64 target_lat			# Info of current and target latitudes and longitudes
float64 target_long
---
#result definition
---
#feedback definition